app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java

```diff
@@ -180,12 +151,40 @@ private void onBasemapLoaded() {
         setOnStatusChangedListener(null);
     }
 
+    private void setExtentOverIsrael() {
+        Envelope israelEnvelope = new Envelope(
```

Comment:
I cant :<
It needs to get the extent to set.
Am I missing something ?
I need 4 points for that  (envelope/4-sided polygon) how can I set the extent from a single point ?